build/setup.d/20-install-packages.sh

```diff
@@ -18,7 +18,6 @@ libswitch-perl
 libwww-perl
 libyaml-dev
 libyaml-0-2
-linux-image-extra-virtual-lts-xenial
```

Comment:
generic should have all of them. If someone depends on something special it would be great to have a test for this.

However afaik it was added while i'm already worked on the branch. Therefor i will give the virtual kernel + extra-virtual another try. Maybe this combination is enough for the nvidia driver